caringcaribou/modules/uds.py

```diff
@@ -1,13 +1,16 @@
+import argparse
+import datetime
+import time
+
+from sys import stdout, stderr
+from typing import List
```

Comment:
This import is redundant, as `list` could be used in type hints instead of `typing.List` (which is deprecated according to https://docs.python.org/3/library/typing.html#typing.List )